metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -395,8 +395,14 @@ def process_dataflow_graph(
         # traverse edges to fill in node properties
         for edge in dataflow_graph["DagEdges"]:
 
-            source_node = nodes[edge["Source"]]
-            target_node = nodes[edge["Target"]]
+            source_node = nodes.get(edge["Source"])
+            target_node = nodes.get(edge["Target"])
+
+            # Currently, in case of unsupported connectors,
+            # Source and Target for some edges is not available
+            # in nodes. this may lead to broken edge in lineage.
+            if source_node is None or target_node is None:
```

Comment:
Should we log any warning here? Is it worth doing?

Reply:
yes, done.